src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -764,29 +766,46 @@ class FlaxBertPreTrainedModel(FlaxPreTrainedModel):
 
     config_class = BertConfig
     base_model_prefix = "bert"
-    module_class: nn.Module = None
+    if typing.TYPE_CHECKING:
+        module_class: ClassVar[Type[nn.Module]]
+    else:
+        module_class = None
 
     def __init__(
         self,
-        config: BertConfig,
+        _config: BertConfig,
+        _module: Any = dataclasses.MISSING,  # unused, defined to comply with dataclass signature
         input_shape: Tuple = (1, 1),
         seed: int = 0,
         dtype: jnp.dtype = jnp.float32,
         _do_init: bool = True,
         gradient_checkpointing: bool = False,
+        *,
+        parent=None,
+        name=None,
```

Comment:
Are these two args required? Do we need them if `FlaxPreTrainedModel` is a nn.Module?